runner/ollamarunner/runner.go

```diff
@@ -946,13 +957,18 @@ func (s *Server) embeddings(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	seq, err := s.NewSequence(req.Content, nil, NewSequenceParams{
 		embedding: true,
-
-		// TODO (jmorganca): this should be provided by the server via the
-		// request options and truncated here in the runner, instead of relying on
-		// the server's truncate logic
-		truncate: true,
+		truncate:  req.Truncate, // always error on over-length; server handles truncation & retry
```

Comment:
Given the comment, should this be set to false? or do we want to leave it as the value of `req.Truncate`?